examples/dialog-modal/datepicker-dialog.html

```diff
@@ -0,0 +1,739 @@
+                </ul>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </section>
+
+      <section>
+          <h3 id="rps_label_3">Date Picker Dialog</h3>
+          <table aria-labelledby="rps_label_3 rps_label" class="data attributes">
+            <thead>
+              <tr>
+                <th scope="col">Role</th>
+                <th scope="col">Attribute</th>
+                <th scope="col">Element</th>
+                <th scope="col">Usage</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr data-test-id="dialog-role">
+                <th scope="row"><code>dialog</code></th>
+                <td></td>
+                <td>
+                  <code>div</code>
+                </td>
+                <td>
+                  <ul>
+                    <li>Identifies <code>div</code> element as a dialog box container.</li>
+                    <li>The accessible name comes from the <code>aria-labelledby</code> attribute.</li>
+                  </ul>
+                </td>
+              </tr>
+              <tr data-test-id="dialog-aria-modal">
+                <td></td>
+                <th scope="row"><code>aria-modal=<q>true</q></code></th>
+                <td><code>div</code></td>
+                <td>
+                  Identifies that dialog box as a modal dialog box.
+                </td>
+              </tr>
+              <tr data-test-id="dialog-aria-labelledby">
+                <td></td>
+                <th scope="row"><code>aria-labelledby=<q>IDREFS</q></code></th>
+                <td><code>div</code></td>
+                <td>
+                  Defines the accessible name for the dialog box.
+                </td>
+              </tr>
+              <tr data-test-id="dialog-aria-live">
+                <th></th>
+                <th scope="row"><code>aria-live=<q>polite</q></code></th>
+                <td><code>div</code></td>
+                <td>
+                  <ul>
+                    <li>A live region to display information about the keyboard commands to navigate the grid of dates.</li>
+                    <li>The display of the information is slightly delayed so the screen reader reads it after information related to the change of focus in the date.</li>
+                  </ul>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </section>
+
+      <section>
+          <h3 id="rps_label_4">Date Picker Dialog: Calendar Navigation Buttons</h3>
+          <table aria-labelledby="rps_label_4 rps_label" class="data attributes">
+            <thead>
+              <tr>
+                <th scope="col">Role</th>
+                <th scope="col">Attribute</th>
+                <th scope="col">Element</th>
+                <th scope="col">Usage</th>
+              </tr>
+            </thead>
+            <tbody>
+                <tr data-test-id="test-not-required">
```

Comment:
Please delete this row. We do not document implied roles.

Reply:
@mcking65 
Removed reference to implied button role